DataManagementSystem/DB/FileCatalogComponents/DatasetManager.py

```diff
@@ -340,7 +347,26 @@ def __getMetaQueryParameters( self, metaQuery, credDict ):
                      'LFNIDList': lfnIDList } )
     return result
 
-  def removeDataset( self, datasetName, credDict ):
+  def removeDataset( self, datasets, credDict ):
+    """ Remove the requested datasets
+
+    :param dict datasets: dictionary describing dataset definitions
+    :param credDict:  dictionary of the caller credentials
+    :return: S_OK/S_ERROR bulk return structure
+    """
+    failed = dict()
+    successful = dict()
+    for datasetName in datasets:
+      result = self.__removeDataset( datasetName )
```

Comment:
self.__removeDataset() takes 2 arguments (the second is credDict). Run pylint on this code and you'll spot these things immediately.